### ISSUE TITLE
Verbosity/check on subscription id

### ISSUE THREAD

**Issue by yucwan** [microsoft/azure-maven-plugins#931]
As part of verbosity improvement, below is an error reported by customer that should be handled with better verbosity.

[ERROR] Failed to execute goal com.microsoft.azure:azure-webapp-maven-plugin:1.8.0:deploy (default-cli) on project friend-with-earth: Parameter this.client.subscriptionId() is required and cannot be null.

**Comment by gkgaurav31** [comment]
I have been facing similar issue when deploying to Azure WebApp: 

mvn package azure-webapp:deploy 

[ERROR] Failed to execute goal com.microsoft.azure:azure-webapp-maven-plugin:1.8.0:deploy (default-cli) on project helloworld: Parameter this.client.subscriptionId() is required and cannot be null. -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal com.microsoft.azure:azure-webapp-maven-plugin:1.8.0:deploy (default-cli) on project helloworld: Parameter this.client.subscriptionId() is required and cannot be null.

How do we get around this error?

**Comment by yucwan** [comment]
@gkgaurav31 It would be great if you can provide more information about the error. For example, did you logged in to an account with valid Azure subscription? which authentication method are you using?

Besides, you can try declaring Subscription id in your pom.xml, like:
```xml
    <plugin>
        <groupId>com.microsoft.azure</groupId>
        <artifactId>azure-webapp-maven-plugin</artifactId>
        <version>1.8.0</version>
        <configuration>
            <!-- Specify v2 schema -->
            <schemaVersion>v2</schemaVersion>
            <!-- App information -->
            <subscriptionId>SUBSCRIPTION_ID</subscriptionId>
            <resourceGroup>RESOURCEGROUP_NAME</resourceGroup>
            <appName>WEBAPP_NAME</appName>
            …
```

**Comment by Flanker32** [comment]
Fixed in #973
